Assets/Scripts/MarchingCubesTunnelGenerator.cs

```diff
@@ -109,6 +128,26 @@ void PopulateVoxels()
             }
         }
 
+        void PopulateVoxelsForCollider(Collider collider)
```

Comment:
`collider` is not used in this method, I intended this method to just check one collider at a time.